components/PralineForm/IngredientList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+
+export default function IngredientList({
+  label,
+  id,
+  onAddIngredient,
+  zutaten,
```

Comment:
This applies to every instance of mixing German into English or vice versa:
Please consider sticking to just English. People who can't speak German wouldn't be able to read your code.